drafts/draft-brockners-ippm-ioam-vxlan-gpe.xml

```diff
@@ -272,7 +277,7 @@
          keywords will be used for the search engine. -->
 
     <abstract>
-      <t>In-situ Operations, Administration, and Maintenance (IOAM) records
+      <t>In Situ Operations, Administration, and Maintenance (IOAM) records
```

Comment:
RFC9197 uses "In situ". Given that you already started to harmonize the spelling by removing the hyphen, could you harmonize to lower case "situ" throughout the doc?

Reply:
Thanks, Frank! [2e667f4](https://github.com/inband-oam/ietf/pull/264/commits/2e667f4cbf702064a873f1129db566b96f8c59d7)